### DISCUSSION TITLE
settings.py empty?

### DISCUSSION THREAD

**Discussion by albert-elmore** [evennia/evennia#2447] [Evennia Questions] [answered]
Should settings.py be empty after installing and running migrate? I do I need to rebuild my install? 

Also some of the in game super user commands seem buggy on my install. For example: 

**Command 'mudinfo' is not available. Maybe you meant "mudinfo" or "find"?**

Should I wipe out this build and start again from the beginning?

**Comment by kovitikus** [comment, marked as answer]
`settings.py` is there for you to override the Evennia default settings, located in the `evennia/evennia/settings_default.py` file.

You just copy and paste over anything you want to change to your project's `settings.py` file and GG.

```
Master configuration file for Evennia.

NOTE: NO MODIFICATIONS SHOULD BE MADE TO THIS FILE!

All settings changes should be done by copy-pasting the variable and
its value to <gamedir>/server/conf/settings.py.

Hint: Don't copy&paste over more from this file than you actually want
to change.  Anything you don't copy&paste will thus retain its default
value - which may change as Evennia is developed. This way you can
always be sure of what you have changed and what is default behaviour.
```

**Reply by albert-elmore** [reply]
Thank you! :)

**Reply by kovitikus** [reply]
Make sure to mark a comment as the answer, so that others know you no longer need assistance. :)

**Comment by Griatch** [comment]
The `mudinfo not found` error is a little confusing and is because `mudinfo` is a channel command and you are not giving an argument. It's a quirk of the way channels work in Evennia 0.9.5 that will be fixed in the next version. There is nothing wrong with your install (and you never need to reinstall to fix problems anyhow).
